client/coral-embed-stream/src/CommentStream.js

```diff
@@ -46,7 +51,8 @@ const mapDispatchToProps = (dispatch) => ({
   },
   appendItemArray: (item, property, value, addToFront, itemType) =>
     dispatch(appendItemArray(item, property, value, addToFront, itemType)),
-  logout: () => dispatch(logout())
+  logout: () => dispatch(logout()),
+  changeTab: activeTab => dispatch(changeTab(activeTab))
```

Comment:
can we just delete this entire `mapDispatchToProps`? These are all just passthroughs and (unless I'm misunderstanding something) are adding needless complexity and room for typos.